exonum-light-client/src/main/java/com/exonum/client/ExplorerApiHelper.java

```diff
@@ -169,36 +140,143 @@ static ExecutionStatus buildExecutionStatus(ErrorKind errorKind, int code,
     @NonNull
     TransactionMessage message;
     TransactionLocation location;
-    JsonObject locationProof; //TODO: in scope of LC P3
-    GetTxResponseExecutionResult status;
+    JsonObject locationProof; // TODO: in scope of LC P3
+    GetTxResponseExecutionStatus status;
   }
 
   /**
    * Json object wrapper for transaction execution result, i.e.,
    * {@code "$.status"}.
+   *
+   * <p>See serde::ExecutionStatus: https://github.com/exonum/exonum/blob/v1.0.0-rc.1/exonum/src/runtime/error/execution_status.rs#L102
```

Comment:
What do you mean by `serde` prefix?

Reply:
The module: there are two ExecutionStatuses, the one which is used for JSON serialization is in module serde